### PR TITLE
Update Ship of Harkinian 2 to v1.1.1

### PR DESCRIPTION
**Pull request by JeodC**
- Update port to v1.1.1 release
- Remove compatibility support

Requires new GLIBC filter.

### CODE REVIEW

File: ports/soh2/README.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ## Information
-2Ship2Harkinian binaries were built from the develop branch (bleeding edge) on 08/14/2024. You can build your own binaries by following the [BUILDING.md guide](https://github.com/JeodC/PortMaster-2Ship2Harkinian/blob/main/BUILDING.md).
+2Ship2Harkinian binaries were built from the develop branch (bleeding edge) on 08/14/2024. You can build your own binaries by following the [BUILDING.md guide](soh2/BUILDING.md).
```
Comment (kloptops) [inline ports/soh2/README.md:2]:
```suggestion
2Ship2Harkinian binaries were built from the develop branch (bleeding edge) on 08/14/2024. You can build your own binaries by following the [BUILDING.md guide](https://github.com/PortsMaster-MV/PortMaster-MV-New/blob/main/ports/soh2/soh2/BUILDING.md).
```

This URL won't work when the README is on the website. :)

**Review by kloptops — CHANGES_REQUESTED** [review]

**Review by kloptops — APPROVED** [review]

**Review by Cebion — CHANGES_REQUESTED** [review]

File: ports/soh2/Ship of Harkinian 2.sh [hunk 2]
```diff
@@ -119,10 +88,11 @@ fi
...
 # Run the game
 echo "Loading, please wait... (might take a while!)" > $CUR_TTY
-pm_platform_helper "2s2h.elf"
 $GPTOKEYB "2s2h.elf" -c "soh2.gptk" & 
 ./2s2h.elf
 
 # Cleanup
 rm -rf "$GAMEDIR/logs/"
-pm_finish
+$ESUDO systemctl restart oga_events &
```
Comment (Cebion) [inline ports/soh2/Ship of Harkinian 2.sh:96]:
```suggestion
pm_finish
```

Comment (Cebion) [inline ports/soh2/Ship of Harkinian 2.sh:96]:
```suggestion
```

File: ports/soh2/Ship of Harkinian 2.sh [hunk 3]
```diff
@@ -119,10 +88,11 @@ fi
...
 echo "Loading, please wait... (might take a while!)" > $CUR_TTY
-pm_platform_helper "2s2h.elf"
 $GPTOKEYB "2s2h.elf" -c "soh2.gptk" & 
 ./2s2h.elf
 
 # Cleanup
 rm -rf "$GAMEDIR/logs/"
-pm_finish
+$ESUDO systemctl restart oga_events & 
+printf "\033c" >> /dev/tty1
```
Comment (Cebion) [inline ports/soh2/Ship of Harkinian 2.sh:97]:
```suggestion
```

File: ports/soh2/Ship of Harkinian 2.sh [hunk 4]
```diff
@@ -119,10 +88,11 @@ fi
...
+printf "\033c" > /dev/tty0
```
Comment (Cebion) [inline ports/soh2/Ship of Harkinian 2.sh:98]:
```suggestion
```

File: ports/soh2/Ship of Harkinian 2.sh [hunk 5]
```diff
@@ -119,10 +88,11 @@ fi
 
 # Run the game
 echo "Loading, please wait... (might take a while!)" > $CUR_TTY
-pm_platform_helper "2s2h.elf"
 $GPTOKEYB "2s2h.elf" -c "soh2.gptk" &
```
Comment (Cebion) [inline ports/soh2/Ship of Harkinian 2.sh:91]:
```suggestion
pm_platform_helper "2s2h.elf"
$GPTOKEYB "2s2h.elf" -c "soh2.gptk" & 
```

**Review by Cebion — DISMISSED** [review]

File: ports/soh2/Ship of Harkinian 2.sh [hunk 6]
```diff
@@ -88,11 +88,10 @@ fi
 
 # Run the game
 echo "Loading, please wait... (might take a while!)" > $CUR_TTY
+pm_platform_helper "2s2h.elf"
```
Comment (kloptops) [inline ports/soh2/Ship of Harkinian 2.sh:91]: pm helper should be after GPTOKEYB
```suggestion
```

File: ports/soh2/Ship of Harkinian 2.sh [hunk 7]
```diff
@@ -88,11 +88,10 @@ fi
...
 $GPTOKEYB "2s2h.elf" -c "soh2.gptk" & 
 ./2s2h.elf
```
Comment (kloptops) [inline ports/soh2/Ship of Harkinian 2.sh:93]:
```suggestion
pm_platform_helper "$GAMEDIR/2s2h.elf"
./2s2h.elf
```

**Review by kloptops — CHANGES_REQUESTED** [review]

**Review by kloptops — APPROVED** [review]
🤞

**Pull request merged**